uwsgi.h

```diff
@@ -1515,8 +1522,9 @@ struct wsgi_request {
 	// when set, do not send warnings about bad behaviours
 	int post_warning;
 
-	size_t range_from;
-	size_t range_to;
+	enum uwsgi_range range_parsed;
+	int64_t range_from;
```

Comment:
No, `size_t` is only for arrays in memory, so it is always 32bit on 32bit platform.
`-D_FILE_OFFSET_BITS=64` will set `off_t` type as 64bit integer.